src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,4 @@
+$days-size: 100px;
+$gap: 1px;
+$padding: 10px;
+$transition: all 0.5s ease;
```

Comment:
Not really wrong I suppose, but a suggestion, and my personal preference, If I'm going to use a style only once, why bother making it a variable?

Reply:
Noted